#request constants
int8 STOP = 0
int8 FOLLOW = 1
int8 LAND = 2
int8 PRECISION_LAND = 3
int8 TAKE_OFF = 4
int8 SET_EKF_SRC = 5

#request fields
int8 mode
float64 altitude # for SET_EKF_SRC this must be an int equal to 1, 2 or 3, depending on which source it should be changed to
---

#response fields
bool success
